docs/USER.md

```diff
@@ -158,6 +159,15 @@ stringData:
 After reconcillation SPIAccessToken should be filled with the Access Token metadata, it's `status.Phase` should be `Injected` and upload Secret is removed.
 In a case if something goes wrong the reason will be written to K8s Event, you can check it with `kubectl get event $upload-secret`.
 
+## Refreshing OAuth Access Tokens
+If a service provider issues OAuth access tokens with an `expiry` time, such tokens need to be refreshed after this time.
+There is a work in progress on an automatic solution. However, the user can manually request a token refresh by
+adding the label `spi.appstudio.redhat.com/refresh-token: true` to SPIAccessToken after the token is in the `ready` phase.
+Note that currently, only GitLab allows refreshing OAuth tokens.
```

Comment:
I would put supported providers at the beginning. Even 1st line to be something like `Supported tokens: Gitlab OAuth`. wdyt ?

Reply:
Fixed.